src/lib/wdi5-control.ts

```diff
@@ -124,18 +124,20 @@ export class WDI5Control {
      * @return {WebdriverIO.Element} the webdriver Element
      */
     async getWebElement() {
+        if (util.types.isProxy(this._domId)) {
+            const id = await Promise.resolve(this._domId)
+            if (id) {
+                const webElement = await $(`//*[@id="${id}"]`)
+                return webElement
+            } else {
+                throw Error("control could not be found")
```

Comment:
could some sort of an identifier be added to the error message to provide an even better aka more fine-tuned feedback?

Reply:
As this is not the only error that will be displayed in the terminal I thought this was sufficient. What is now displayed with the current change:

```zsh
[0-0] [wdi5] call of _getControl() failed because of: Error: No DOM element found using the control selector {"id":"test","properties":{"text":"open Dialog"},"viewName":"test.Sample.view.Main","sOriginalControlType":"sap.m.Button"}
[0-0] [wdi5] error retrieving control: testtestSample.view.Mainsap.m.Button{"text":"open Dialog"}
[0-0] [wdi5] Can not call press(), because control could not be found
```
What do you think can be described here more detailed?

Reply:
thanks for the context and sample output → 👍